atlas_loader/mount_loader.lua

```diff
@@ -291,7 +291,7 @@ function M.load(info)
     if M.use_html_loader and html_loader then
         html_loader.show()
     end
-    if not liveupdate then
+    if not liveupdate or not liveupdate.is_built_with_excluded_files() then
```

Comment:
Maybe it would be better to add a check for the existence of the `liveupdate.is_built_with_excluded_files` function?
Otherwise, 1.9.6 and previous versions will not work